scripts/classification/imagenet/verify_pretrained.py

```diff
@@ -56,12 +59,19 @@
 
 normalize = transforms.Normalize([0.485, 0.456, 0.406], [0.229, 0.224, 0.225])
 
-transform_test = transforms.Compose([
-    transforms.Resize(256, keep_ratio=True),
-    transforms.CenterCrop(224),
-    transforms.ToTensor(),
-    normalize
-])
+if 'inceptionv3' in model_name:
+    transform_test = transforms.Compose([
```

Comment:
inception also needs a resize before center crop: https://github.com/tensorflow/models/blob/master/research/inception/inception/image_processing.py#L294.

Note that although the tf doc says `central_fraction` is the fraction of area, it is actually the fraction of both sides according to its code: https://github.com/tensorflow/tensorflow/blob/r1.10/tensorflow/python/ops/image_ops_impl.py#L611

Reply:
@hetong007 , appreciate for your time on reviewing and detailed explaination, I noticed the central fraction of 224 case is 0.875(=224/256), though seems the nvcaffe handle 299 case by 320-crop, I therefore choose 320, what do you think? Is 320 okay, or I should follow the 0.875 as TF chosing.

Reply:
I don't either find your code to resize input to `320`. I suggest we use 0.875 for consistency with the original repository.

Reply:
Resize to 342 this time to keep 0.875, changes now applied to both raw images input and rec images. 
See line 115~123 and line 62 to line 68. Thanks again for CR.